### PR TITLE
[Feat-759] Add version number to Footer

### PR DESCRIPTION
**Pull request by segevda**

### CODE REVIEW

File: src/components/organisms/Footer.tsx [hunk 1]
```diff
@@ -7,38 +7,36 @@ import { Typography } from 'components/atoms';
 import { silverSmokeColor, oceanBlueColor, skyBlueColor } from 'style';
 import ThankYouDialog from './ThankYouDialog';
 import AboutDialog from './AboutDialog';
+import { version } from '../../../package.json';
```
Comment (daniel-ilan) [inline src/components/organisms/Footer.tsx:10]: no need to import relatively, you can just do this `import { version } from 'package.json'`

**Review by daniel-ilan — APPROVED** [review]

File: src/components/organisms/Footer.tsx (hunk 1 above)
Comment (segevda) [reply src/components/organisms/Footer.tsx:10]: this doesn't work for me. as far as I understand, absolute paths are available under paths listed as `include` in `tsconfig.json` ([here](https://create-react-app.dev/docs/importing-a-component/#absolute-imports)). Am I missing something?

Comment (yuvalbl) [reply src/components/organisms/Footer.tsx:10]: you can use absolute imports - `/package.json`

**Review by yuvalbl — APPROVED** [review]

File: src/components/organisms/Footer.tsx (hunk 1 above)
Comment (segevda) [reply src/components/organisms/Footer.tsx:10]: as discussed, this requires changes to `tsconfig.json` or creating a symlink to `package.json` from within `/src`. For now, leaving the relative path as is

**Pull request merged**